openbas-front/src/admin/components/atomic_testings/atomic_testing/AtomicTesting.tsx

```diff
@@ -125,6 +131,37 @@ const AtomicTesting = () => {
                     }
                   />
                 </ListItemButton>
+                <List component="div" disablePadding>
+                  {target?.children?.map((child) => <Paper elevation={1} style={{ marginBottom: 5 }} key={child?.id}>
+                    <ListItemButton
+                      sx={{ pl: 6 }}
+                      key={child?.id}
+                      onClick={() => handleTargetClick(child)}
+                    >
+                      <ListItemText
+                        primary={
+                          <div>
+                            <div style={{
+                              color: 'gray',
+                              display: 'inline-block',
+                              float: 'left',
+                              paddingRight: 10,
+                            }}
+                            >{getIcon(child?.targetType)}</div>
+                            <div className={classes.bodyTarget} style={{ width: '30%' }}>
+                              {`${child?.name}`}
+                            </div>
+                            <div style={{ float: 'right' }}>
+                              <AtomicTestingResult
+                                expectations={child?.expectationResultsByTypes}
+                              />
+                            </div>
+                          </div>
```

Comment:
Maybe, you can write a component in a const to re use it.
I saw this line two times :)